tests/admin_inlines/tests.py

```diff
@@ -1262,6 +1262,11 @@ class BothVerboseNameProfileInline(TabularInline):
         self.assertNotContains(response, '<h2>Model with both - plural name</h2>')
         self.assertNotContains(response, 'Add another Model with both - name')
 
+    def test_immutable_inlines_on_base_model_admin(self):
+        modeladmin = ModelAdmin(ProfileCollection, admin_site)
+        with self.assertRaises(AttributeError):
+            modeladmin.inlines.append(InnerInline)
```

Comment:
I don't see much value in this test.

Reply:
@felixxm I don't really have an opinion here. If you want to drop it 👍